kuma/wiki/models.py

```diff
@@ -1478,6 +1481,11 @@ def get_contributors(self):
         return User.objects.filter(pk__in=list(top_creator_ids))
 
 
+@receiver(render_done)
+def purge_wiki_url_cache(sender, instance, **kwargs):
+    memcache.delete(u'wiki_url:%s:%s' % (instance.locale, instance.slug))
```

Comment:
Would it make sense to set the cache here instead of just deleting it? This works but I'm just curious.